go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/openfaas-incubator/faas-memory
+
+go 1.19
```

Comment:
This should be go 1.18 to match the rest of OpenFaaS. We will get to 1.19, but need more time.

Whatever value is set here is a minimum version for any code that imports provided packages.

Reply:
fixed :)